projects/example-app/src/app/auth/actions/login-page.actions.ts

```diff
@@ -0,0 +1,28 @@
+import { Action } from '@ngrx/store';
+import { User, Authenticate } from '../models/user';
+
+export enum LoginPageActionTypes {
+  Login = '[Login Page] Login',
+  LoginFailure = '[Login Page] Login - Failure',

```

Comment:
`LoginFailure` and `LoginSuccess` aren't needed here. They success/fail is covered by the Auth API actions.